pageserver/src/repository.rs

```diff
@@ -11,9 +11,17 @@ use zenith_utils::lsn::Lsn;
 /// A repository corresponds to one .zenith directory. One repository holds multiple
 /// timelines, forked off from the same initial call to 'initdb'.
```

Comment:
The fork point is not necessary the same initial initdb, we can branch at any moment/timeline, right? Well, I guess this says initdb is the deepest common ancestor of everyone...

Reply:
> Well, I guess this says initdb is the deepest common ancestor of everyone...

Right. Although we've also chatted about allowing importing completely unrelated postgres clusters to the same page server or repository. Nothing in the page server depends on all timelines sharing a common ancestor, there just isn't any UI to do that currently.